sle15/profiles/stig.profile

```diff
@@ -238,6 +238,8 @@ selections:
     - sudo_remove_nopasswd
     - sudo_restrict_privilege_elevation_to_authorized
     - sudo_require_authentication
+    - sudoers_validate_passwd
+    - susefirewall2_only_required_services
```

Comment:
This line is not related to the rule, what is its purpose?

Reply:
@yarunachalam line susefirewall2_only_required_services was removed from sle15. So, it should be removed.

Reply:
removed